Xamarin.Forms.Core.Design/Properties/AssemblyInfo.cs

```diff
@@ -4,7 +4,4 @@
 [assembly: AssemblyTitle ("Xamarin.Forms.Core.Design")]
 [assembly: AssemblyDescription ("Provides the design-time metadata for the XAML language service.")]
```

Comment:
All these attributes can be configured via MSBuild properties instead in SDK-style projects, see https://github.com/dotnet/sdk/blob/master/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.GenerateAssemblyInfo.targets#L67-L96

Reply:
The design projects are not sdk style yet